src/audio.c

```diff
@@ -544,6 +546,17 @@ static void poll_aubuf_tx(struct audio *a)
 		sampc = sampc_rs;
 	}
 
+	/* check if we should generate DTMF audio */
+	if (tx->enc_fmt == AUFMT_S16LE &&
+	    dtmf_is_empty(&tx->dtmfgen) == false) {
```

Comment:
comparing with false is not needed, better to write it like this:

`!dtmf_is_empty`